### PR TITLE
Moved ColorSliderGroup to FC.

### PR DESCRIPTION
**Pull request by NitzanWix**
## Description
Moved ColorSliderGroup to FC.

### CODE REVIEW

File: src/components/slider/ColorSliderGroup.tsx [hunk 1]
```diff
@@ -42,79 +42,82 @@ export type ColorSliderGroupProps = {
...
+    sliderContainerStyle
+  } = props;
+  const [color, setColor] = useState(initialColor);
+  useEffect(() => {
+    setColor(initialColor);
+  }, [initialColor]);
 
-  state = {
-    initialColor: this.props.initialColor
+  const onValueChange = (value: string) => {
```
Comment (Inbal-Tish) [inline src/components/slider/ColorSliderGroup.tsx:99]: Please wrap in useCallback

File: src/components/slider/ColorSliderGroup.tsx [hunk 2]
```diff
@@ -42,79 +42,82 @@ export type ColorSliderGroupProps = {
...
-    }
-    return null;
+  const sliderProps = {
+    initialColor,
+    sliderContainerStyle,
+    showLabels,
+    labelsStyle,
+    accessible,
+    labels,
+    migrate
```
Comment (Inbal-Tish) [inline src/components/slider/ColorSliderGroup.tsx:110]: Seems like you could pick up the props you need for the group (container) and pass the others as "others" to the ColorSlider, no?

Comment (NitzanWix) [reply src/components/slider/ColorSliderGroup.tsx]: Most of them. Yeah. Fixed.

File: src/components/slider/ColorSliderGroup.tsx [hunk 3]
```diff
@@ -42,79 +42,37 @@ export type ColorSliderGroupProps = {
...
-    prevState) => {
-    if (prevState.initialColor !== nextProps.initialColor) {
-      return {
-        initialColor: nextProps.initialColor
-      };
-    }
-    return null;
-  };
+  const onValueChange = useCallback((value: string) => {
+    _.invoke(props, 'onValueChange', value);
```
Comment (Inbal-Tish) [inline src/components/slider/ColorSliderGroup.tsx:64]: use optional chaining instead of lodash's invoke

**Comment by Inbal-Tish** [comment]
@NitzanWix Looks great. I left a small coment

File: src/components/slider/ColorSliderGroup.tsx [hunk 4]
```diff
@@ -42,79 +41,37 @@ export type ColorSliderGroupProps = {
...
-    prevState) => {
-    if (prevState.initialColor !== nextProps.initialColor) {
-      return {
-        initialColor: nextProps.initialColor
-      };
-    }
-    return null;
-  };
+  const onValueChange = useCallback((value: string) => {
+    props.onValueChange?.(value);
```
Comment (Inbal-Tish) [inline src/components/slider/ColorSliderGroup.tsx:63]: props can potentially be undefied so add a ? after

Comment (NitzanWix) [reply src/components/slider/ColorSliderGroup.tsx]: Fixed

**Review by Inbal-Tish — APPROVED** [review]

**Pull request merged**